### PR TITLE
Release 3.5 merge

### PR DESCRIPTION
**Pull request by rprouse**
This is ready to merge after review. This includes the changes made for the 3.5 framework release and it increases the version to 3.6 for ongoing development.

Delete the branch once merged, the release branch is separate.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,4 +1,51 @@
...
+may diverge over time.
+
+This is also the first release where the NUnit Framework will not be included in the installer. Only
+the console runner, engine and extensions will be available as an MSI installer. We recommend that you
+use the NUnit NuGet packages for the framework, but a ZIP file with the binaries will also be available.
+
+Framework
+
+ * Added Assert.Zero and Assert.NotZero methods
+ * Tests with a Timeout will no longer timeout while you are debugging
```
Comment (CharliePoole) [inline CHANGES.txt]: I usually put any feature changes likely to require documentation updates here and then use it to drive the docs updates. By that criterion, I think #144 and #1553 qualify.

Comment (CharliePoole) [inline CHANGES.txt]: Obviously, we can't change the copy that's in the release, but we can still update the online release notes and fix CHANGES.txt going forward.

**Comment by kloun** [comment]
all is ok.

File: CHANGES.txt (hunk 1 above)
Comment (rprouse) [inline CHANGES.txt]: I will make the changes tonight and push them before we merge this PR.

**Comment by kloun** [comment]
Also I wrote news for LOR:  https://www.linux.org.ru/news/opensource/12925207

**Pull request merged**